### ISSUE TITLE
SPE and THREE r74

### ISSUE THREAD

**Issue by RemusMar** [squarefeet/ShaderParticleEngine#91]
Hi Luke,

Any chance to make SPE compatible with the latest THREE release (74) ? 

cheers

**Comment by RemusMar** [comment]
I was able to sort them up (many syntax changes), but the old bug is back:
https://github.com/squarefeet/ShaderParticleEngine/issues/72 
Long story short:
If you change the "emitter.size.value" at runtime, the value is NOT changed for the current particles.
Only for the upcoming ones!

**Comment by squarefeet** [comment]
Hey @RemusMar,

What changes did you make to support r74? I'm awaiting the changelog for this release before making any changes to SPE. 

I'll look into the size issue but can't guarantee anything!

**Comment by RemusMar** [comment]
Hi Luke,

Not a single change in the current SPE !!!
I found solutions for everything.
As for changing the emiter.size.value at runtime, that is not needed anymore.
I found another approach with perfect results:
		group.scale = group.uniforms.scale.value = 300 * 'runtime value';

http://necromanthus.com/Test/html5/testA.html 
Take a look at the "steam" sources and press the right mouse button a few times (for zoom in & out).
cheers
